src/peer-discovery/tests/index.js

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
why the no check ?

Reply:
I am excluding the interface tests for now. This is not a priority and we should probably tackle these together with the actual modules implementations to have type checks there